### ISSUE TITLE
Typos in Web Property Bag properties for MasterPage inheritance

### ISSUE THREAD

**Issue by orty** [OfficeDev/PnP#1296]
Hi,
There are two typos in the following class : `OfficeDevPnP.Core\AppModelExtensions\BrandingExtensions.cs` which introduce bugs while provisioning the branding of SharePoint sites.

Here are the actually defined properties :
```
const string InheritMaster = "__InheritMasterUrl";
const string InheritCustomMaster = "__InheritCustomMasterUrl";
```

But they should be written this way :
```
const string InheritMaster = "__InheritsMasterUrl";
const string InheritCustomMaster = "__InheritsCustomMasterUrl";
```

Btw, thanks a lot for your huge work !

**Comment by VesaJuvonen** [comment]
thx Serge. We'll get this tested and fixed.

**Comment by orty** [comment]
Thanks you :)

**Comment by patrick-rodgers** [comment]
Fixed this with  [PR#341](https://github.com/OfficeDev/PnP-Sites-Core/pull/341)
